memorious/logic/meta.py

```diff
@@ -0,0 +1,24 @@
+# from typing_extensions import TypedDict
```

Comment:
IMO this is not generic enough to be its own module. We should put it in the aleph operation module.

Reply:
But it's not a module. It's the definition of a type. I would type definitions seperate from modules that actually do stuff.